src/client/sandboxed-jquery.js

```diff
@@ -1719,7 +1719,11 @@ export function init (window, undefined) {
                     container.style.zoom = 1;
                 }
 
-                body.removeChild(container);
+                // NOTE: in some cases Firefox can't remove container div,
```

Comment:
I don't think so. The comment in the issue contains the same info I've written above and doesn't contain any additional info